linked_list/linked_list.py

```diff
@@ -23,13 +23,31 @@ def __init__(self):
     self.tail = None
 
   def add_to_tail(self, value):
-    pass
+    if self.tail is None:
+      first_node = Node(value)
+      self.head = first_node
+      self.tail = first_node
+    else:
+      newest_node = Node(value)
+      self.tail.next_node = newest_node
+      self.tail = newest_node
+      self.tail.next_node = None
```

Comment:
Great implementation here, seems to work alright. When you use `newest_node`, are you referring to the node after the original head?